charts/nexus-iq/templates/deployment.yaml

```diff
@@ -35,6 +35,8 @@ spec:
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           env:
+            - name: SONATYPE_WORK
```

Comment:
Adding `SONATYPE_WORK` env variable to be able to use it on `iq.env` config option in `values.yaml`